transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -810,6 +822,10 @@ public final void flush() {
                 return;
             }
 
+            flush(outboundBuffer);
```

Comment:
It would seem that ignoring explicit flushing would also go some way to addressing [comment 1 from @buchgr ](https://github.com/netty/netty/pull/5395#issuecomment-229290268)

@buchgr @Scottmitch thoughts?